src/Tomlyn/Text/CharHelper.cs

```diff
@@ -24,6 +24,11 @@ public static bool IsControlCharacter(char32 c)
             return c <= 0x1F || c == 0x7F;
         }
 
+        public static bool IsNonNewlineValidControlCharacter(char32 c)
+        {
+            return c == '\b' || c == '\t' || c == '\f';
```

Comment:
Don't think that `\b` and `\f` are allowed as per the [specs](https://toml.io/en/v1.0.0#string)

> Any Unicode character may be used except those that must be escaped: quotation mark, backslash, and the control characters other than tab (U+0000 to U+0008, U+000A to U+001F, U+007F).

Reply:
I think this line is talking in double quote context, with double quote string, we have to escape control characters except tab. But in literal string, we can use them as is.
Because it's a normal use case where one copies pastes from other document to toml literal string and so we may often encounter `\f`.

Reply:
Actually now looking at the spec again, I think we should do it like this:
* **Single line string literal**: allow any control character except newline
* **Multiline string literal**: allow any control character, newline included

Reply:
> I think this line is talking in double quote context, with double quote string, we have to escape control characters except tab. But in literal string, we can use them as is.

Sorry, I missed that it was in the context of literal string.

Though, a little bit down the specs, about literal string, they mention this:

> Control characters other than tab are not permitted in a literal string.

Reply:
OK, I found that line. I'll make another commit.